src/app/components/layout/navbar/navbar.component.ts

```diff
@@ -456,9 +458,15 @@ export class NavbarComponent implements OnInit, AfterViewInit, OnDestroy {
   }
 
   navToServiceConfiguration(service) {
+    let isServiceEnabled = ["shutdown", "disabled", "installed"].includes(service.status) ? false : true;
+    if (['failed', 'unresponsive'].includes(service.status.toLowerCase())) {
+      let isScheduleEnabled = this.schedulesData.find(sch => sch.name === service.name)?.enabled;
+      isServiceEnabled = ["shutdown", "disabled", "installed"].includes(service.status) || !isScheduleEnabled ? false : true;
```

Comment:
how disabled or installed set enabled TRUE?

Reply:
Please check again, in these cases `isServiceEnabled ` set to **false**

Reply:
My concern is why do we care to check them; default should be set to false then set to true if you see it running|schedule enabled

Reply:
Yes, We can do in that way also.

Reply:
Why do we have repeated logic and cannot have a common code to know this!

Reply:
We have already minimised the code as much as we can. From here we are sending data to a centralised code (component) where we are managing all the logic.